### DISCUSSION TITLE
Build the Future of AI Autonomy: Join Formi's Mission to Transform Business Intelligence

### DISCUSSION THREAD

**Discussion by AryamannNingombam** [community/community#147432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

At Formi, we’re creating a context-aware, observation-driven agentic layer that serves as a proactive companion for businesses and individuals. Our agents operate seamlessly across platforms and modalities, functioning autonomously without the need for prompts or triggers to take action.

We have over 80 large customers like - Embassy, Royal Orchid hotels and Barbeque Nation.

Your Mission (If You’re Ready for It): 

We’re not hiring you for the usual. These are not roles for anyone looking to play it safe. We need someone who’s ready to challenge the status quo and make radical changes. You’ll be building the future with us, so expect to be pushed—and to push back. We’re not here to settle for less.

These Roles are NOT for You If:
- You’re just looking for another job.
- You think “good enough” is, well, good enough.
- You’d rather play it safe than take bold, calculated risks.
- We’re hiring for some exciting roles and are looking for freshers and early-career professionals who are eager to learn, grow, and make an impact.

Software Engineer - AI Systems Generalist
[https://forms.gle/BRaix1nwTLwdNUGD9](https://forms.gle/BRaix1nwTLwdNUGD9)
Software Engineer, AI Evaluation Specialist
[https://forms.gle/K8dFpoRLqK2EzAcf6](https://forms.gle/K8dFpoRLqK2EzAcf6)

Inquiries:
Mail: hv@formi.co.in
WhatsApp: 6361826089

**Comment by shinybrightstar** [comment]
Hi USERNAME, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
